packages/cli/test/unit/util/input/playground.test.ts

```diff
@@ -0,0 +1,246 @@
+import { describe, it, expect } from 'vitest';
+import { client } from '../../../mocks/client';
+import { isWindows } from '../../../helpers/is-windows';
+
+const theme = {
+  // Override spinner with a single frame
+  spinner: { frames: ['O'] },
+};
+
+// Inquirer's unicode detection is outdated, for windows in CI
+// it incorrectly detects that unicode is not supported,
+// and thus prints ascii characters, which breaks these tests.
+// https://github.com/SBoudrias/Inquirer.js/issues/1386
+describe.skipIf(isWindows)('client.input', () => {
```

Comment:
Just skipping these on Windows

Reply:
Thanks for the comment in code about this. Makes sense.